samples/nvidia-resnet/trtis_client/demo_client_ui.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.

```

Comment:
Is there a way to add this visualization as an output of the pipeline so that the user does not need to run a new webserver? (The Kubeflow/Pipeline UI can display static HTML).

Reply:
We can definitely use the pipeline to start a client server but that's probably not the best design because the client server (which hosts the webapp) doesn't have to be in the same system as the inference server (which serves the model). In addition, our demo client is dynamic so not sure if the Pipeline UI is a good fit.

Reply:
It might be possible. We'll look into it.

Reply:
We've integrated the webapp component to the pipeline itself. In addition, the webapp UI is routed to a subpath of Kubeflow UI (e.g. localhost:[kubeflow-port]/webapp/)